### ISSUE TITLE
Verify email template wrong again

### ISSUE THREAD

**Issue by agjohnson** [readthedocs/ext-theme#518]
![image](https://github.com/user-attachments/assets/f9ff7230-fbfe-442e-9127-d836636bd396)

The header should be smaller.

**Comment by humitos** [comment]
By the way, the wrong template (the one from the description) is the one shown at https://app.readthedocs.com/accounts/confirm-email/#/email immediately after signing up.

The correct template is showing at https://app.readthedocs.com/accounts/confirm-email/<hash>/ when clicking on the link from the email

![Screenshot_2025-01-15_11-53-13](https://github.com/user-attachments/assets/839841ff-1caf-42c1-b08a-f18f0f8e752d)

**Comment by humitos** [comment]
> the wrong template (the one from the description) is the one shown at https://app.readthedocs.com/accounts/confirm-email/#/email immediately after signing up.

I wasn't able to reproduce this locally on Community at least.

However, accessing this URL manually http://devthedocs.org/accounts/confirm-email/#/email I can see the wrong template. The file used is `/.venv/lib/python3.12/site-packages/allauth/templates/account/verification_sent.html`

**Comment by agjohnson** [comment]
Yeah I've been a bit lost on the verification system lately too, it seems there are a few new pieces to Allauth verification that we weren't using or didn't exist previously. I don't remember how I triggered this either, it might have been with email sign up though.
